domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -554,11 +554,19 @@ private static void streamTrailer(HttpServerResponse response, ResultInfo result
     String element, RoutingContext routingContext, Map<String, String> okapiHeaders,
     Context vertxContext) {
 
+   streamGet(table,clazz,cql,offset,limit,facets,element,routingContext,okapiHeaders,vertxContext,0);
+  }
+
+   public static <T> void streamGet(String table, Class<T> clazz,
```

Comment:
Can you add javadoc for all parameters?
Can you add javadoc what unit (minutes, seconds, ...) the timeout variable has?

Reply:
Can you add
```
@SuppressWarnings({"squid:S107"})     // Method has >7 parameters
```
to suppress the warning that sonar raises? https://sonarcloud.io/project/issues?id=org.folio%3Araml-module-builder&pullRequest=657&resolved=false&types=CODE_SMELL